extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -431,8 +431,33 @@ export const popupMessageListener = (request: Request, sessionStore: Store) => {
     return { publicKey };
   };
 
+  /* 
+  The user has lost their password and they want to reimport an account.
+  Remove all references to the previous account in localStore so the user has a clean slate to start over.
+  */
+  const _removePreviousAccount = async () => {
+    await localStore.remove(ACCOUNT_NAME_LIST_ID);
+    await localStore.remove(APPLICATION_ID);
+    await localStore.remove(RECENT_ADDRESSES);
+    await localStore.remove(LAST_USED_ACCOUNT);
+    await localStore.remove(TOKEN_ID_LIST);
+
+    const keyIdList = await getKeyIdList();
+
+    for (let i = 0; i < keyIdList.length; i += 1) {
+      const k = keyIdList[i];
+      await localStore.remove(`stellarkeys:${k}`);
```

Comment:
is this the list of account stellar addresses the user has created?

Reply:
like in the format `stellarkeys:GCGP37AHTWZXS7SRWEP6KNXB4U4U3D473BL3WKAPGXBBDIOGGCA367YP`

Reply:
Yup, that's mostly correct:

These are the encrypted blobs that are created by `@stellar/typescript-wallet-sdk-km`. The format is `stellarkeys:{keyID}`. So, for ex, it might be `stellarkeys:0.987383812837182`

Reply:
aah, got it. So it's actually encrypted there. Good to know, thanks for the clarification!